app/views/responses/_error.xml.builder

```diff
@@ -0,0 +1,6 @@
+xml.instruct! :xml, version: '1.0'
+xml.tag!('OAI-PMH', oai_pmh_header) do
```

Comment:
The header stuff, which is going to be reused across partials, is probably best put into a layout that then yields for the internal contents, so we can reuse it everywhere instead of doing it in every partial. 

Let me know if you want a bit of guidance on how to do this.

Reply:
Done, the only thing is that there are different headers for different responses (haven't noticed a pattern yet, looks like there are at least two variants)